tests/phpunit/tests/db.php

```diff
@@ -48,6 +48,18 @@ public function query_filter( $sql ) {
 		return $sql;
 	}
 
+	/**
+	 * Test Version number is accessible
+	 *
+	 * Make sure $wpdb->version is available for plugins to check what features are available.
+	 *
+	 * @ticket 52506
+	 */
+	public function test_version() {
+		global $wpdb;
```

Comment:
Need to document the global in the docblock.

Reply:
Thanks, done